balancer-js/src/modules/pools/pool-types/concerns/stable/join.concern.ts

```diff
@@ -15,6 +15,24 @@ import { AddressZero } from '@ethersproject/constants';
 import { BalancerError, BalancerErrorCode } from '@/balancerErrors';
 import { StablePoolEncoder } from '@/pool-stable';
 import { _upscaleArray } from '@/lib/utils/solidityMaths';
```

Comment:
I think its useful to order imports. Have the external package imports at the top followed by internal functions then test imports.

Reply:
Not sure you followed what I described? I meant things like ethers, other packages first, then SDK related imports like slippage helpers then at bottom test related imports.
Let me know if you think there's a good reason to do this differently.